commcare_connect/opportunity/tests/test_api_views.py

```
@@ -260,28 +261,62 @@ def test_delivery_progress_endpoint(
+        assert response.status_code == 200
+        response = api_client.get(f"/api/opportunity/{opportunity.id}/delivery_progress")
+        assert response.status_code == 200
+        payment_data = response.data["payments"][0]
+        assert payment_data["confirmed"] is False
+
+    def test_confirm_payment_for_nm_org(
+        self,
+        program_manager_org_user_admin: User,
+        program_manager_org: Organization,
+        api_client: APIClient,
+        opportunity: Opportunity,
```

Ruff (0.14.8)

Unused method argument: `opportunity`

(ARG002)